docker/puppetserver-standalone/webserver.conf

```diff
@@ -0,0 +1,6 @@
+webserver: {
+    access-log-config: /etc/puppetlabs/puppetserver/request-logging.xml
+    client-auth: want
+    ssl-host: 0.0.0.0
+    ssl-port: ${PUPPET_MASTERPORT}
```

Comment:
Does this ENV variable get resolved? It might; they don't in pe-console-services at least.

Also, I noticed this webserver.conf file is modified via `hocon` in the entrypoint script: https://github.com/puppetlabs/puppetserver/pull/2078/files#diff-f9e912c5472d58a4ee06a97dc9592516R31

Just pointing that out since I hadn't seen that before.

Reply:
Ah, good call about using `hocon` to manage webserver.conf, I've updated!